src/sentry/web/frontend/accounts.py

```diff
@@ -37,7 +38,8 @@ def login_redirect(request):
 
 
 def expired(request, user):
-    password_hash = LostPasswordHash.for_user(user)
+    password_hash = LostPasswordHash.objects.for_user(user)
+    LostPasswordNotification(password_hash, ip=request.META["REMOTE_ADDR"]).send()
```

Comment:
@mgaeta This PR is already changing 35 files, that's a lot. Can we add the new notifications in a subsequent PR instead?